.github/workflows/unit-tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: ['3.10']
```

Comment:
bug in GitHub actions requiring a string [here](https://github.com/actions/runner/issues/1989)